errors/errors.go

```diff
@@ -121,9 +129,31 @@ func CreateEndpointNotEnabled(param string) Error {
 	}
 }
 
-// UnauthorizedError when an operation is not permitted.
+// UnauthorizedError when access to the service is disallowed
 type UnauthorizedError struct{}
 
 func (e UnauthorizedError) Error() string {
 	return "UnauthorizedError: Unauthorized"
 }
+
+func concat(values ...string) string {
```

Comment:
Why implementing our own concat function. Can't we use https://pkg.go.dev/strings#Join ?